setup.py

```diff
@@ -16,18 +16,27 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-# pylint: disable=invalid-name
+# pylint: disable=invalid-name,wrong-import-position
+
+import sys
+
+from pathlib import Path
 
 from setuptools import setup, find_packages
 
-version = __import__('gvm').get_version()
+__here__ = Path(__file__).parent.resolve()
+
+sys.path.insert(0, str(__here__))
+
+from gvm import get_version
+
 
-with open('README.md', 'r') as f:
+with (__here__ / "README.md").open("r") as f:
```

Comment:
Why is this an improvement? Seems more complicated.

Reply:
Without this change
```sh
some/path$ python path/to/setup.py build_wheel
```
doesn't work. And using pathlib instead of os.path seems better to me with python3. I guess it could even be more simplified with pathlib but i didn't take a deeper look at the pathlib api.